mixs-templates/README.md

```diff
@@ -1,11 +1,17 @@
 ## MIxS (meta)data collection spreadsheet templates
 
-This folder contains the MIxS schema (meta)data collection templates in the Excel spreadsheet (.xlsx) format. Each of the templates in this folder are blank spreadsheet templates with only the header row populated with the names of the terms that are associated with a particular [checklist](https://sujaypatil96.github.io/mixs/#checklists), [extension](https://sujaypatil96.github.io/mixs/#extensions), or [combination](https://sujaypatil96.github.io/mixs/combinations/) (of one checklist plus one extension). The templates offer pulldown menus for filling out columns for which there is a brief [enumeration](https://sujaypatil96.github.io/mixs/enumerations/) of permissible values, like [window_type](https://sujaypatil96.github.io/mixs/0000856/), but this [feature is disabled](https://linkml.io/linkml/generators/excel.html) if the total character length of the permissible values is more than 255, like [fao_class](https://sujaypatil96.github.io/mixs/0001083/).
+In this folder, the GSC provides Microsoft Excel (.xlsx) spreadsheets for collecting metadata that is roughly aligned with the MIxS standard. THe GSC appreciates that spreadsheets like these can be very useful for preparing data to be submitted to an INSDC database. Users should be aware of the limitations of these spreadsheets, listed below.
```

Comment:
Questions:
* what does "roughly aligned" mean?
* typo "THe" --> "The"

Reply:
thanks @sujaypatil96 I really appreciate this review and apologize for not replying sooner

I think (!) "roughly aligned" means that the templates provide prompts (in the form of column headers) to use the terms associated with the template's class, but it doesn't do anything to ensure that valid values are provided. I'll fix that up.